doc/10-icinga-template-library.md

```diff
@@ -2259,6 +2259,42 @@ snmp_service_count         | **Optional.** Compare matching services with a spec
 snmp_service_showall       | **Optional.** Show all services in the output, instead of only the non-active ones. Defaults to false.
 snmp_service_noregexp      | **Optional.** Do not use regexp to match NAME in service description. Defaults to false.
 
+### elasticsearch (NETWAYS) <a id="plugin-check-command-elasticsearch"></a>
```

Comment:
This should not be a subsection of "Plugin Check Commands for Manubulon SNMP". It should rather be next to the existing `elasticsearch` command.